appveyor.yml

```diff
@@ -0,0 +1,60 @@
+---
+version: '{branch}-{build}'
+skip_tags: true
+max_jobs: 1
+image: Visual Studio 2013
+init:
+  - git config --global core.autocrlf input
+# Uncomment the following line to show RDP info at beginning of job
+#  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+clone_folder: c:\projects\mixxx
+configuration:
+  - release
+#  - debug
+environment:
+  matrix:
+    - platform: x64
+      bindir: dist64
+    - platform: x86
+      bindir: dist32
+matrix:
+  fast_finish: false     # set this flag to true to immediately finish build once one of the jobs fails.
+cache:
+  - C:\mixxx-buildserver
+install:
+  - cd %TEMP%
+  - echo *** Downloading and installing scons
+  # SF.net download currently broken from EC2 to some mirrors, so hosting scons myself
+#  - curl -fsS -L -o scons-2.5.1.zip http://prdownloads.sourceforge.net/scons/scons-2.5.1.zip
+  - curl -fsS -L -o scons-2.5.1.zip http://url_to_appveyor_build_env/scons-2.5.1.zip
```

Comment:
It's kind of a long shot but someone could tamper with the files via man-in-the-middle over HTTP. Given that we're going to produce builds for users to install via this process, I would prefer using HTTPS here to prevent tampering.